### PR TITLE
Box layout broken in description box while add new sub category #24352

### PR DESCRIPTION
**Pull request by rajeshwebkul**
### Description (*)
Box layout broken in description box while add new sub category

### Fixed Issues (if relevant)

1. magento/magento2#24352: Box layout broken in description box while add new sub category

### Manual testing scenarios (*)

Go to Admin
Catalog -> Categories
Add Subcategory
Content -> Description

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/design/adminhtml/Magento/backend/Magento_Catalog/web/css/source/_module.less [hunk 1]
```diff
@@ -102,3 +102,13 @@
         }
     }
 }
+
+@media screen and (min-width: 1080px) {
```
Comment (dmytro-ch) [inline app/design/adminhtml/Magento/backend/Magento_Catalog/web/css/source/_module.less:106]: Please use the default [Magento 2 breakpoints](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.html#fedg_rwd_css_break).
The fix is not related to the `Magento_Catalog` module only. So, please add it to the appropriate LESS file.
e.g. `app/design/adminhtml/Magento/backend/web/css/source/forms/_form-wysiwyg.less`.

File: app/design/adminhtml/Magento/backend/Magento_Catalog/web/css/source/_module.less [hunk 2]
```diff
@@ -102,3 +102,13 @@
...
+    .page-layout-admin-2columns-left {
+        .admin__control-wysiwig {
+            iframe {
+                width: 99.9% !important;
```
Comment (dmytro-ch) [inline app/design/adminhtml/Magento/backend/Magento_Catalog/web/css/source/_module.less:110]: Please avoid using the force state "!important" whenever it's possible.

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hi @sagarwebkul,
the current solution is rather a workaround 
```
width: 99.9%
```
and it does not seem to be solving the real issue.
I would recommend investigating the issue deeper and solving the reason of the bug.

**Comment by VladimirZaets** [comment]
@sagarwebkul, I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**